.github/workflows/ci.yml

```diff
@@ -5,7 +5,15 @@ on:
   - pull_request
 
 jobs:
+  ruby-versions:
+    uses: ruby/actions/.github/workflows/ruby_versions.yml@master
+    with:
+      engine: cruby
+      min_version: 2.5
+      versions: "[debug]"
```

Comment:
Perhaps like this:

```suggestion
      versions:
        - debug
```

Reply:
Thank you!

Reply:
It seems that we can't use JSON array here...: https://github.com/ruby/fiddle/actions/runs/7472168103/workflow

```text
The workflow is not valid. .github/workflows/ci.yml (Line: 14, Col: 9): A sequence was not expected
```

Reply:
thanks. fixed [6e2f4b9](https://github.com/ruby/fiddle/pull/136/commits/6e2f4b9957c2aa167bf8924bb33373a71a200fdf)